### DISCUSSION TITLE
식별자란 무엇인가요?

### DISCUSSION THREAD

**Discussion by publdaze** [publdaze/js-study#16] [변수]
- 식별자란 무엇인가요?

**Comment by pipisebastian** [comment]
> 식별자란 무엇인가요?

식별자는 변수의 이름입니다. 즉, 메모리 주소에 붙인 이름이며
변수뿐만 아니라 함수, 클래스의 이름 모두 식별자입니다.

**Comment by publdaze** [comment]
> - 식별자란 무엇인가요?
- 변수, 함수, 클래스 등의 이름을 식별자라고 함
- 어떤 값을 구별해서 식별할 수 있는 이름
